securemailbox/constants.py

```diff
@@ -0,0 +1,4 @@
+# Declare any variables (as constants) that are or could be used anywhere in the application
+
+# TODO: Determine what a good fingerprint length is
+FINGERPRINT_LENGTH = 100
```

Comment:
Now, GPG uses 40 hex digits (160 bits SHA-1) fingerprint. 100 is too long.
The OpenPGP specification (which contains GPG) includes the [description](https://tools.ietf.org/html/rfc4880#section-12.2) of the length of a fingerprint.

Reply:
Perfect thanks!